.github/workflows/release.yml

```diff
@@ -46,6 +46,9 @@ jobs:
     runs-on: ubuntu-latest
     needs:
       - weekindex
+    strategy:
+      matrix:
+        repo: ['nasa-impact/veda-config', 'US-GHG-Center/veda-config-ghg']
```

Comment:
I don't think this will work - because dispatching an event to GHG-Center will require a different auth token.